eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -39,6 +39,7 @@ jobs:
           dotnet pack eng/service.proj -warnaserror
           /p:ServiceDirectory=${{ parameters.ServiceDirectory }}
           /p:IncludeTests=false
+          /p:ExcludeProjects="${{ join('|', parameters.ExcludeProjects) }}"
```

Comment:
Though the expression is used in a couple of call sites, I chose not to move this call into a variable;  my goal was to be explicit and clear on how the value is formed, since there is string parsing taking place in the project file.  I feel that having the join inline improves locality and prevents having to scan the file to understand what is happening.